packages/web-components/fast-foundation/src/text-field/text-field.spec.ts

```diff
@@ -199,6 +199,66 @@ describe("TextField", () => {
 
         await disconnect();
     });
+    it("should hide the label when start and/or end content is provided", async () => {
+        const { element, connect, disconnect } = await setup();
+        const content: HTMLImageElement = document.createElement("img");
+        content.src = "https://placehold.it/16";
```

Comment:
Could this cause timeout errors when running in an environment without a network connection?

Reply:
probably, It really can be anything so I'll change it.